modules/sdk/gradle-plugins-defaults/CHANGELOG.markdown

```diff
@@ -2004,7 +2004,8 @@ plugin is no longer applied to OSGi projects.
 ## 5.0.30 - 2017-08-29
 
 ### Added
-- [LPS-73070]: Check and keep version in the `package-lock.json` file in sync.
+- [LPS-73070]: Check the module's version in the `package-lock.json` file and
+ensure it matches the `Bundle-Version` in the `bnd.bnd`.
```

Comment:
I would just say "...and ensure that it matches the project version", since we have themes that have a `package-lock.json` but not a `bnd.bnd`. "project version" is a Gradle concept.

Reply:
@Ithildir good point. I'll make that change.